### DISCUSSION TITLE
part-ii/2021-19/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#248] [Question comments]
Part II 2021 Applied Probability: Paper 4, Section II, $27 \mathrm{~K}$ 

https://questions.tripos.org/part-ii/2021-19/

**Comment by DannyDuberstein** [comment]
Any hints for the first part?

**Reply by eha1234** [reply]
Write out the expectation as a sum then differentiate using the product rule :)

**Reply by DannyDuberstein** [reply]
A sum over j of p_{ij}(t) E_i [ f(...) | X(t) = j] doesn't seem easy to differentiate?

**Reply by eha1234** [reply]
You can differentiate $p_{ij}$ using KBE equations. Then conditional on $X(t) = j, f(...) = f(j, l+L(t))$. Also $L_j(t)$ is just increasing linearly and all other $L_i$ are constant, if that makes sense?
